CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.5.3
```

Comment:
If we're looking to do a release, it looks like a number of things have been merged since 1.5.2: https://github.com/puppetlabs/trapperkeeper/compare/1.5.2...master